bcda/auth/plugin/alpha.go

```diff
@@ -68,7 +68,36 @@ func (p *AlphaAuthPlugin) DeleteClient(params []byte) error {
 
 // can treat as a no-op or call RequestAccessToken
 func (p *AlphaAuthPlugin) GenerateClientCredentials(params []byte) ([]byte, error) {
-	return nil, errors.New("not yet implemented")
+	var empty []byte
```

Comment:
It might be a little more readable to call this `responseBody` or something along those lines, then setting it to equal ``fmt.Sprintf(`{"tokenString":"%s"}`, tokenString)`` when successful.

Reply:
I thought of that as well, but that seems to run against the go gestalt of nil-ish response. We may also change the signature of the methods, so let's defer this conversation until then.